### PR TITLE
Add missing generated option to documentation

### PR DESCRIPTION
**Pull request by alexander-schranz**
Seems not be documented in: https://github.com/doctrine/orm/pull/9118

### CODE REVIEW

File: docs/en/reference/basic-mapping.rst [hunk 1]
```diff
@@ -239,6 +239,7 @@ Here is a complete list of ``Column``s attributes (all optional):
 - ``nullable`` (default: ``false``): Whether the column is nullable.
 - ``insertable`` (default: ``true``): Whether the column should be inserted.
 - ``updatable`` (default: ``true``): Whether the column should be updated.
+- ``generated`` (default: ``null``): Whether the generated strategy should be ``'NEVER'``, ``'INSERT'`` and ``ALWAYS``.
```
Comment (alexander-schranz) [inline docs/en/reference/basic-mapping.rst:242]: not sure if we should document `NEVER` as for me it doesn't make lot of sense if default is null doing nothing. But it was added to PHPDoc: https://github.com/doctrine/orm/blob/cfc0655a1c9b5ffd5506172cb7642b6090aa5f7e/src/Mapping/Column.php#L18

/cc @mehldau

Comment (SenseException) [reply docs/en/reference/basic-mapping.rst:242]: `NEVER` is a valid value so it should be mentioned. A default may change over time.

**Review by SenseException — DISMISSED** [review]

**Comment by greg0ire** [comment]
Please kindly squash your commits together. If you don't, we'll try to remember to do it for you but it's best if you save us this trouble.

How to do that?

1. `git rebase -i origin/2.19.x`, assuming `origin` is a git remote that points to this repository, and not your fork. If you're not sure what your remotes are, run `git remote -vvv`, there should be your fork and the holy/reference/base/origin/whatever-you-call-it repository.
2. A window will show up with many lines, replace `pick` with `fixup` on every line but the first one
3. Close your editor, git should do its magic, and you should end up with one commit
4. Use `git push --force` to overwrite what you already push. Don't forget the `--force` option otherwise git will try to merge both things together.

**Comment by alexander-schranz** [comment]
@greg0ire done. PS you can also enable squash or rebase merge in the Github UI in the Settings to merge a pull request via squash or rebased methods.

**Comment by greg0ire** [comment]
I know but again…

> we'll try to remember to do it for you but it's best if you save us this trouble.

I cannot guess how you are going to combine the commit messages together.

**Review by greg0ire — APPROVED** [review]

**Comment by greg0ire** [comment]
Thanks @alexander-schranz

**Pull request merged**